src/commands/install.ts

```diff
@@ -30,43 +42,92 @@ export default class Install extends Command {
     }),
   };
 
-  private async installSingle(service_id: string) {
-    const service_dir = process.cwd();
-    const parts = service_id.split(':');
-    const service_name = parts[0];
-    let service_tag = parts[1];
-    const config = ServiceConfig.loadFromPath(service_dir);
+  async run() {
+    await this.tasks();
+  }
 
-    if (config.name === service_name) {
-      throw new Error('Services cannot depend on themselves');
-    }
+  async tasks() {
+    const { args, flags } = this.parse(Install);
+    const root_service_path = flags.prefix ? flags.prefix : process.cwd();
+    const dependency_graph = await generateGraphFromPaths([process.cwd()], new EnvironmentConfigV1(), this.app.api);
```

Comment:
`process.cwd()` should be `root_service_path`, but also see my prior comment about allowing multiple paths to be passed in.